src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTestsBase.cs

```diff
@@ -43,9 +45,9 @@ namespace Microsoft.NET.Sdk.Razor.SourceGenerators;
 [Collection(nameof(RazorSourceGenerator))]
 public abstract class RazorSourceGeneratorTestsBase
 {
-    protected static async ValueTask<GeneratorDriver> GetDriverAsync(Project project)
+    protected static async ValueTask<GeneratorDriver> GetDriverAsync(Project project, Action<TestAnalyzerConfigOptionsProvider>? configureGlobalOptions = null)
     {
-        var (driver, _) = await GetDriverWithAdditionalTextAsync(project);
+        var (driver, _) = await GetDriverWithAdditionalTextAsync(project, configureGlobalOptions);
```

Comment:
Part of test infra changes from https://github.com/dotnet/razor/pull/8520 to allow setting RazorLangVersion in tests.